test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -664,7 +668,7 @@ private static JComponent createInstructionUIPanel(String instructions,
         text.setEditable(false);
 
         JPanel textPanel = new JPanel(new BorderLayout());
-        textPanel.setBorder(createEmptyBorder(4, 0, 0, 0));
+        textPanel.setBorder(createEmptyBorder(GAP, 0, GAP, 0));
         textPanel.add(new JScrollPane(text), BorderLayout.CENTER);
```

Comment:
I accidentally committed these three lines in ![21162](https://git.openjdk.org/jdk/pull/21162) for [JDK-8340785](https://bugs.openjdk.org/browse/JDK-8340785) and I didn't notice it.

This three-line snippet was completely unrelated to updating description of the `PassFailJFrame` class and samples.